### ISSUE TITLE
Borg tool slots becomiong unusable when deconstructed into a basic borg

### ISSUE THREAD

**Issue by McRex007** [tgstation/tgstation#29530]
I made myself into a janitorial borg, went around for a bit, and decided I didn't like that so I went back to robotics to get myself reverted. I didn't remove the tools I had in my quickslots before this. Once I specialized as an engineering borg I couldn't equip to any slots, and I couldn't store anything from any of the slots, which were displaying as if they were empty.

**Comment by AnonymousNow** [comment]
Did you try de-equipping your current equipment as if you had invisible things in your hands? I found that sometimes after a changed cyborg equipment state it would think I'm still holding onto something invisible.

**Comment by Anonmare** [comment]
Yeah use the PageUp button to select the active modules and then de-quip them. The tools are still equipped, their sprites are just invisible

**Comment by Supermichael777** [comment]
how odd that deff shouldn't happen
